config/routes.rb

```diff
@@ -3,7 +3,7 @@
   resources :merchants do
 
     resources :invoices, controller: 'merchant_invoices'
-    resources :items, controller: 'merchant_items', only: [:index, :show]
+    resources :items, controller: 'merchant_items', only: [:index, :show, :edit, :update]
```

Comment:
At this point we might want to move to an , except[:destroy, etc.]

Reply:
Yea if we're using basically everything else that might be a good idea